python/kserve/kserve/handlers/v2_datamodels.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, List, Union, Dict
+
+import orjson
+from pydantic import BaseModel
+
+# TODO: in the future, this file can be auto generated
```

Comment:
This file contains the Pydantic data models for V2 Inference protocol, which can be used by FastAPI to generate OpenAPI schema.
We also added a few examples here which will be rendered in the Swagger UI.